cloud-controller-manager/do/kubernetes_lb_validator.go

```diff
@@ -121,15 +126,15 @@ func (v *KubernetesLBServiceValidator) Handle(ctx context.Context, req admission
 	v.Log.Info("validating create request")
 	resp, err = v.validateCreate(ctx, lbRequest)
 	if err != nil {
-		validError, errorCode := v.isValidationError(resp)
-		if !validError || errorCode != http.StatusUnprocessableEntity {
-			v.Log.Error(err, "failed to validate lb creation, could not get validation response")
-			return admission.Errored(int32(errorCode), errors.Wrap(err, "failed to validate lb create, could not get validation response"))
+		errorCode := getStatusCode(resp)
+		if errorCode != http.StatusUnprocessableEntity {
+			v.Log.Error(err, "failed to validate lb creation")
+			return admission.Errored(int32(errorCode), errors.Wrap(err, "failed to validate lb create"))
 		}
-		v.Log.Error(err, "failed to validate lb creation, invalid configuration")
-		return admission.Denied(fmt.Sprintf("failed to validate lb creation: %v", err))
+		v.Log.Error(err, "invalid LB update configuration")
+		return admission.Denied(fmt.Sprintf("invalid LB update configuration: %v", err))
 	}
-	v.Log.Info(fmt.Sprintf("allowing create"))
+	v.Log.Info("allowing create")
```

Comment:
nit: this says "allowing create" while in the upgrade path we say "lb update validated". A consistent wording would probably read slightly better.